docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -0,0 +1,54 @@
+# Stanford Basic theme
+
+Stanford Basic is used a base for all future H&S themes. It integrates selectively with Decanter Sass, styles and 
```

Comment:
We have pulled from patterns and templates in Stanford Basic (stanford_basic) as well as SU Humsci (su_humsci_theme) Themes.